libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/ConfigService.java

```diff
@@ -114,11 +112,14 @@ private String formatPath(String path, String... pathProperties) {
         return String.format(path, (Object[]) pathProperties);
     }
 
-    // PYIC-7048 Replace this with proper config
     public long getBearerAccessTokenTtl() {
-        return Optional.ofNullable(getEnvironmentVariable(BEARER_TOKEN_TTL))
-                .map(Long::valueOf)
-                .orElse(DEFAULT_BEARER_TOKEN_TTL_IN_SECS);
+        try {
+            return Long.parseLong(getParameter(ConfigurationVariable.BEARER_TOKEN_TTL));
+        } catch (ConfigParameterNotFoundException | NumberFormatException e) {
+            LOGGER.warn(
+                    LogHelper.buildErrorMessage("Failed to get bearer token ttl from config", e));
+            return DEFAULT_BEARER_TOKEN_TTL_IN_SECS;
+        }
     }
```

Comment:
I think I'd prefer removing this method entirely, and either doing `Long.parseLong(getParameter(...))` where it's used, or adding a more generic `getLongParameter(..)` which does the parsing for you.

As a general rule, I think we should avoid adding specific methods for each bit of config (or we should add specific methods for _all_ config values, but that's very tedious).